src/Components/Sticky/Sticky.tsx

```diff
@@ -27,18 +27,19 @@ import { useNavBarHeight } from "../NavBar/useNavBarHeight"
  *      If a selector to a target is specified (for example, bottomBoundary="#footer"),
  *      the offset will be the bottom of the target
  */
-export const Sticky: React.FC<Pick<ReactStickyProps, "bottomBoundary">> = ({
-  children,
-  bottomBoundary,
-}) => {
+export const Sticky: React.FC<
+  Pick<ReactStickyProps, "bottomBoundary"> & {
+    withoutHeaderOffset?: boolean
+  }
+> = ({ children, bottomBoundary, withoutHeaderOffset }) => {
   const { offsetTop, registerSticky, deregisterSticky } = useSticky()
   const { desktop, mobile } = useNavBarHeight()
   const isMobile = __internal__useMatchMedia(themeProps.mediaQueries.xs)
   const [stuck, setStuck] = useState(false)
   const containerRef = useRef<HTMLDivElement | null>(null)
-  const headerOffset = isMobile ? mobile : desktop
+  const headerOffset = withoutHeaderOffset ? 0 : isMobile ? mobile : desktop
 
-  useEffect(() => {
+  React.useEffect(() => {
```

Comment:
**suggestion:** maybe we should also import this from React as we do for `useRef` and `useState`

Reply:
Done 👍